bcda/models/models.go

```diff
@@ -52,8 +53,9 @@ type JobKey struct {
 type ACO struct {
 	gorm.Model
 	UUID     uuid.UUID `gorm:"primary_key; type:char(36)" json:"uuid"` // uuid
-	Name     string    `json:"name"`                                   // name
-	ClientID string    `json:"client_id"`                              // software client id
+	CMSID    *string   `gorm:"type:char(5); unique" json:"cms_id"`
+	Name     string    `json:"name"`      // name
```

Comment:
do these comments do anything in documentation or otherwise? for consistency, maybe we should remove them all or add a similar one to `CMSID`

Reply:
I was wondering that about the `json` ones. I don't know where we'd be using those because they're not included in our Swagger models, and we wouldn't be unmarshalling ACO JSON objects. I'll try removing.